invenio_administration/views/base.py

```diff
@@ -376,6 +377,9 @@ def get(self):
                 "pid_path": self.pid_path,
                 "create_ui_endpoint": self.get_create_view_endpoint(),
                 "list_ui_endpoint": self.get_list_view_endpoint(),
+                "resource_name": self.resource_name
```

Comment:
In my opinion this creates clutter inside a dictionary's implementation. I don't expect to see a conditional statemente inside a `dict`. 

I suggest you extract this variable before `return self.render`. e.g: 

```python
serialized_schema = self._schema_to_json(schema)
resource_name = self.resource_name or self.id_path

return self.render(...)
```